projects/packages/jetpack-mu-wpcom/src/features/wpcom-themes/wpcom-themes.php

```diff
@@ -87,3 +87,16 @@ function wpcom_themes_remove_wpcom_actions() {
 	);
 }
 add_action( 'load-themes.php', 'wpcom_themes_remove_wpcom_actions' );
+
+/**
+ * Adds a CSS file to fix UI issues in the theme browser.
+ */
+function wpcom_themes_load_ui_fixes() {
+	wp_enqueue_style(
```

Comment:
I'm unable to reproduce the original issue on Atomic sites, so I wonder if we actually need to load this file on them.

Reply:
Seems something on my end, since apparently there are reports of the issue happening on Atomic sites too,